ouroboros-network/test/Test/Pipe.hs

```diff
@@ -24,17 +24,18 @@ import           Test.Tasty.QuickCheck (testProperty)
 
 import           Control.Tracer (nullTracer)
 
+import qualified Network.Mux as Mx
+import qualified Network.Mux.Interface as Mx
+import qualified Network.Mux.Bearer.Pipe as Mx
+
 import           Ouroboros.Network.Chain (Chain, ChainUpdate, Point)
 import qualified Ouroboros.Network.Chain as Chain
 import qualified Ouroboros.Network.ChainProducerState as CPS
-import qualified Ouroboros.Network.Mux as Mx
-import qualified Ouroboros.Network.Mux.Interface as Mx
-import           Ouroboros.Network.Pipe
 import           Ouroboros.Network.Protocol.ChainSync.Client as ChainSync
 import           Ouroboros.Network.Protocol.ChainSync.Codec as ChainSync
 import           Ouroboros.Network.Protocol.ChainSync.Examples as ChainSync
 import           Ouroboros.Network.Protocol.ChainSync.Server as ChainSync
-import qualified Test.Mux as Mxt
+-- import qualified Test.Mux as Mxt
```

Comment:
Remove?

Reply:
Yes, and all this should go to `mux` package too.

Reply:
It cannot it is using `ChainSync` protocol.